### PR TITLE
Brother MFC6490CW driver

### LINKED ISSUE #84779: Brother MFC6490CW
**Project description**

I'm trying to use my printer: a Brother MFC6490CW so I'm trying to package its driver.
I'm looking at examples here: 
https://github.com/NixOS/nixpkgs/tree/master/pkgs/misc/cups/drivers
But I have no idea how they were made so it's not working out great.

**Metadata**

* homepage URL: https://support.brother.com/g/b/producttop.aspx?c=us_ot&lang=en&prod=mfc6490cw_all

### PR DESCRIPTION
**Pull request by NorfairKing**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

I volunteer to maintain this package as long as I have this printer and use NixOS.

This is my first printer driver and also my first real nixpkgs contribution.
Please be nice, but give a lot of feedback.

### CODE REVIEW

**Comment by grahamc** [comment]
@ofborg eval

File: pkgs/misc/cups/drivers/brother/mfc6490cw/default.nix [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  filterFile = "lpd/filter${model}";
+
+in rec {
+  driver = pkgsi686Linux.stdenv.mkDerivation rec {
+    inherit version;
+    src = fetchurl {
+      url = "https://download.brother.com/welcome/dlf006180/${model}lpr-${version}.i386.deb";
+      sha256 = "1p33aal3dv3r03v9nbafrzx2bhij34hrsg2d71kwm738a8n18vnj";
+    };
+    name = "${model}drv-${version}";
```
Comment (jraygauthier) [inline pkgs/misc/cups/drivers/brother/mfc6490cw/default.nix:31]:
```
name = "${model}lpr-${version}"
```

To reflect upstream package name and the name exposed in the all-packages.nix module.

**Review by jraygauthier — CHANGES_REQUESTED** [review]
I like this single file packaging. I see this is a hybrid in between the `pkgs/misc/cups/drivers/brother/mfcl3770cdw/default.nix` single nix file and other lpr / cupswrapper multi nix files packages.

Only issue is with the lpr package name. Also, surround bash variable expansion with double quotes.

**Comment by NorfairKing** [comment]
@jraygauthier Thanks for the review!

**Review by jraygauthier — APPROVED** [review]
LGTM

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -8420,4 +8420,10 @@
     github = "cpcloud";
     githubId = 417981;
   };
+  syd = {
+    name = "Tom Sydney Kerckhove";
+    email = "syd@cs-syd.eu";
+    github = "NorfairKing";
+    githubId = 3521180;
+  };
```
Comment (veprbl) [inline maintainers/maintainer-list.nix:8428]: The new entries should not go to the end of maintainer-list.nix. Please put them in their respective place according to the alphabetic ordering.

**Review by veprbl — COMMENTED** [review]
Please rebase to fix merge conflict in maintainer-list.nix.

**Comment by SuperSandro2000** [comment]
Closing due to inactivity from author.

**Pull request closed without merging**